index.rst

```diff
@@ -238,6 +238,11 @@ described in the previous paragraph is to prevent the need of the additional
 operation of specifically retrieving the filename, which can be relatively
 expensive in the context of EditorConfig.
 
+Empty EditorConfig Files
+------------------------
+An empty ``.editorconfig`` file is considered to be a valid ``.editorconfig`` file 
+with no sections and with implicit ``root`` key set to ``false``.
```

Comment:
How about moving this right before "No Inline Comments"?

What do you think about the following fomulation:

```suggestion
An empty ``.editorconfig`` file contains only Blank and Comment lines, or no line. It is equivalent to
an ``.editorconfig`` file that contains only one line, ``root = true``.
```

Reply:
@xuhdev this seems reasonable to me.  I think there's one typo in your suggestion --- it should be `root = false`, not `true`.

Reply:
Right, `root = false`. Nice catch!

Reply:
I agree with you guys @cxw42 @xuhdev, thanks!

Reply:
Done!